docs/extra.css

```diff
@@ -1,3 +1,9 @@
 .wy-nav-top, .wy-side-nav-search {
     background: #439b47;
 }
+
+.md-typeset__scrollwrap {
+    margin: 1em -.8rem;
+    overflow-x: visible;
+    touch-action: auto;
+}
```

Comment:
some of the changes are coming from the docs change PR, could you rebase and I look again? otherwise looks good

Reply:
Yep, I did rebase. Not sure why this is coming in as a delta.

Reply:
Done.